js/widgets/pitchtimematrix.js

```diff
@@ -577,18 +577,24 @@ function PitchTimeMatrix () {
 
     this._createaddcolumnpiesubmenu = function() {
         docById('wheelDivptm').style.display = '';
-        var valueLabel = ['pitch', 'hertz', 'drum', 'arc', 'forward', 'right', 'setcolor'];
-        this._pitchWheel = new wheelnav('wheelDivptm', null, 600, 600);
+        var valueLabel = ['pitch', 'hertz', 'drum', 'arc', 'graphics'];
```

Comment:
arc is a graphics command

Reply:
we can use just one wheel for 2-arg blocks (e.g., arc) and 1-arg blocks (e.g. forward)